mautrix/bridge/matrix.py

```diff
@@ -349,11 +372,31 @@ async def handle_message(self, room_id: RoomID, user_id: UserID, message: Messag
                 # Not enough values to unpack, i.e. no arguments
                 command = text
                 args = []
-            await self.commands.handle(room_id, event_id, sender, command, args, message, portal,
-                                       is_management, bridge_bot_in_room)
+
+            try:
+                await self.commands.handle(room_id, event_id, sender, command, args, message,
+                                           portal, is_management, bridge_bot_in_room)
+            except Exception as e:
+                await bail(repr(e), step=MessageSendCheckpointStep.COMMAND)
```

Comment:
This is going to double-log the error, but maybe that's fine

Reply:
Where else is it logged?

Reply:
https://github.com/mautrix/python/blob/v0.11.1/mautrix/bridge/commands/handler.py#L412-L413 (a few lines above the `raise` you added)

Reply:
I think that's fine. The log in `handler.py` has some different information, and a stack trace.